lib/debuglet.js

```diff
@@ -142,8 +142,8 @@ Debuglet.prototype.scheduleRegistration_ = function(seconds) {
 
   function onError(err) {
     that.logger_.error('Failed to re-register debuggee: ' + err);
-    that.logger_.error('Disabling gcloud debuglet');
-    that.stop(); // fatal error
+    that.scheduleRegistration_(Math.min((seconds + 1) * 2,
```

Comment:
Let's just make this simpler and omit the exponential backoff. The default value of 40 is small enough. WDYT?